utilities/update_docs_version.sh

```diff
@@ -10,7 +10,7 @@ RELEASED_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate
 if [ "${RELEASED_VERSION##*-}" != "SNAPSHOT" ]; then
     echo "Changing version to $RELEASED_VERSION in README files"
     # Get list of directories for which README.md must be updated
-    module_folders=($(find . -maxdepth 1 -name 'gcloud-java*' -type d) .)
+    module_folders=($(find . -maxdepth 2 \( -regex "./gcloud-java[-[a-z]+]+*" -o -regex "./gcloud-java-contrib/gcloud-java[-[a-z]+]+*" \) -type d) . ./gcloud-java)
```

Comment:
Hm, when I try those two same lines, I get
```
./gcloud-java-dns ./gcloud-java-contrib ./gcloud-java-contrib/gcloud-java-nio ./gcloud-java-storage ./gcloud-java-examples ./gcloud-java-datastore ./gcloud-java-bigquery ./gcloud-java-pubsub ./gcloud-java-resourcemanager ./gcloud-java-compute ./gcloud-java-core . ./gcloud-java
```
It seems like the find returning an empty result.  I've probably used something platform dependent (are you using mac by any chance?), but I imagine that's okay because our Travis builds are on Debian.